### PR TITLE
Fix Matomo dimension IDs

### PR DESCRIPTION
**Pull request by Gudahtt**
The IDs for the `numberOfTokens` and `numberOfAccounts` Action Dimensions were swapped. These are the correct IDs. We'll have to consider that these are swapped when viewing any pre-v8.0.0 metrics.

### CODE REVIEW

File: ui/app/helpers/utils/metametrics.util.js [hunk 1]
```diff
@@ -62,8 +62,8 @@ const customDimensionsNameIdMap = {
   [METAMETRICS_CUSTOM_ENVIRONMENT_TYPE]: 6,
   [METAMETRICS_CUSTOM_ACTIVE_CURRENCY]: 7,
   [METAMETRICS_CUSTOM_ACCOUNT_TYPE]: 8,
-  [METAMETRICS_CUSTOM_NUMBER_OF_TOKENS]: 9,
-  [METAMETRICS_CUSTOM_NUMBER_OF_ACCOUNTS]: 10,
+  [METAMETRICS_CUSTOM_NUMBER_OF_TOKENS]: 10,
+  [METAMETRICS_CUSTOM_NUMBER_OF_ACCOUNTS]: 9,
```
Comment (whymarrh) [inline ui/app/helpers/utils/metametrics.util.js:66]: Could we sort these?

```suggestion
  [METAMETRICS_CUSTOM_NUMBER_OF_ACCOUNTS]: 9,
  [METAMETRICS_CUSTOM_NUMBER_OF_TOKENS]: 10,
```

**Review by whymarrh — CHANGES_REQUESTED** [review]
One small nitpick

**Comment by whymarrh** [comment]
What's the canonical source for these IDs?

File: ui/app/helpers/utils/metametrics.util.js (hunk 1 above)
Comment (Gudahtt) [reply ui/app/helpers/utils/metametrics.util.js]: Done, and I updated the order of the constants to reflect this order.

**Comment by Gudahtt** [comment]
The canonical source for these IDs is our Matomo account, which shows these under the settings (probably only for admins unfortunately).

<details>
<summary>Screenshot:</summary>

action_dimensions

</details>

**Review by whymarrh — APPROVED** [review]
LGTM!

**Pull request merged**